File: parlib_soptpure/parlib/net/inetwork.py
from parlib.net.inetworkreader import INetworkReader

class INetwork:
	def connect(self, host, port):
		return False
	
	def sendMessage(self, message):
		return False
	
	def createNetworkReader(self):
		return INetworkReader();
	
	def setTimeout(self, timeoutmillis):
		return False

File: experiments/proj_ftpclient/jclient_legacy/paskma/ftpclient/net/INetwork.java
package paskma.ftpclient.net;

/**
 * User: paskma
 * Date: Oct 10, 2011
 * Time: 12:44:01 PM
 */
public interface INetwork {
	public boolean connect(String host, int port);
	public boolean sendMessage(String message);
	public INetworkReader createNetworkReader();
}
